nautobot/docs/core-functionality/ipam.md

```diff
@@ -1,9 +1,5 @@
 # IP Address Management
 
-{%
-    include-markdown "../models/ipam/aggregate.md"
-    heading-offset=1
-%}
 {%
```

Comment:
Might consider adding a `--- 2.0.0` comment to this document noting that the Aggregate model was removed? Or is that covered in this page by your notes added to `prefix.md` and `rir.md` do you think?

Reply:
This file is just embedding all of the other files so I don't think it would belong here. I believe it's covered with the changes in `prefix.md`, `rir.md` and `migrating-from-netbox.md`